README.md

```diff
@@ -5,6 +5,8 @@
 
 Drone plugin for deploying to Cloud Foundry
 
+See [DOCS.md](DOCS.md) for `.drone.yml` usage
```

Comment:
But I don't have a strong opinion on that single part if it gets added to all official plugins.